# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/ffvpx/libavutil/avstring.h
# Commit: 186e2a334f2d
# Full Hash: 186e2a334f2deb37b29f44b28bc8c83babeeb9a4
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-09 09:37:17
# Regressor Bug: 1877221
# File Overlap Count: 1
# Description:
#   Bug 1877221 - Pull in new ffmpeg code, reapply patch. r=media-playback-reviewers,chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D199909
# ==============================================================================

diff -r c5920de78d16 -r 186e2a334f2d media/ffvpx/libavutil/avstring.h
--- a/media/ffvpx/libavutil/avstring.h	Thu Feb 08 10:39:53 2024 +0000
+++ b/media/ffvpx/libavutil/avstring.h	Thu Feb 08 10:39:54 2024 +0000
@@ -24,7 +24,6 @@
 #include <stddef.h>
 #include <stdint.h>
 #include "attributes.h"
-#include "version.h"
 
 /**
  * @addtogroup lavu_string
@@ -265,7 +264,7 @@
 
 /**
  * Locale-independent strings replace.
- * @note This means only ASCII-range characters are replace
+ * @note This means only ASCII-range characters are replaced.
  */
 char *av_strireplace(const char *str, const char *from, const char *to);
 